Readme.md

```diff
@@ -1,21 +1,146 @@
+
+The following graphic shows how an exemplary DAO setup, where the
+
+![An examplary DAO setup](https://devs.aragon.org/assets/images/dao-plugin.drawio-7086d0911d25218097dae94665b1a7b1.svg)
+
+An examplary DAO setup showing interactions between the three core contract pieces triggered by different user groups: The `DAO` and `PermissionManager` contract in blue and red, respectively, as well as two `Plugin` contracts in green. Function calls are visualized as black arrows and require permission checks (red, dashed arrow). In this example, the permission manager determines whether the token voting plugin can execute actions on the DAO, a member can change its settings, or if an DeFi-related plugin is allowed to invest in a certain, external contract.
+
+### Framework Contracts
+
+In contrast, the *Framework Contracts* are in charge of creating and registering DAOs and plugins. Additionally, it’s what holds the plugin installer to install or uninstall plugins into DAOs.
+
+- __Factories and Registries__
+    - **The DAO Factory**: In charge of deploying instances of a new DAO based on the parameters given, including which plugins to install and additional metadata the DAO has (like a name, description, etc).
+    - **The DAO Registry**: In charge of registering DAOs into our protocol so we can easily access all DAO instances within our protocol. It is also in charge of giving DAOs subdomains for easier access.
+    - **The Plugin Repo Factory**: A `PluginRepo` is the repository of versions for a given plugin. The Plugin Repo Factory contract creates `PluginRepo` instances so that plugins can update their contract versioning without complexity in a semantic way similar to the App Store.
+    - **The Plugin Repo Registry**: In charge of registering the `PluginRepo` instances so that we easily access all protocol plugins.
+- __Plugin Processor__: installs and uninstalls plugins into DAOs based on the instructions provided by the plugin setup.
+
+For a more detailed description of each of these components, please visit our [Developer Portal](https://devs.aragon.org).
+
+### Plugins
+
+Each plugin consists of two key components:
+
+- __The Plugin Logic__: contains the logic for each plugin; the main functionality the plugin extends for the DAO. Can be linked to other helper contracts if needed.
+- __The Plugin Setup__: contains the installation, uninstallation, and upgrade instructions for a plugin into a DAO.
+
+You can find all plugins built by the Aragon team [here](https://github.com/aragon/osx/tree/develop/packages/contracts/src/plugins).
+
+### Connection between OSx, subgraph, and ethers.js packages
+
+The [Aragon OSx contracts](https://github.com/aragon/osx/tree/develop/packages/contracts) emits events that get indexed within our `subgraph`. This `subgraph`, whose [source code can be found here](https://github.com/aragon/osx/tree/develop/packages/subgraph), is what then fuels the [Aragon SDK](https://github.com/aragon/sdk).
```

Comment:
`aragonOSx` and `aragonSDK`

Reply:
Resolved in Discord :) https://discord.com/channels/672466989217873929/1077542094224236594/1078336009546252350